.github/workflows/pr.yaml

```
@@ -0,0 +1,90 @@
+name: PR checks
+on: pull_request
+
+jobs:
+  run-image:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build image
+        run: DOCKER_BUILDKIT=1 docker build --tag anbox-cloud-io .
+
+      - name: Run image
+        run: |
+          docker run --detach --env SECRET_KEY=insecure_dev_key --network host anbox-cloud-io
+          sleep 1
+          curl --head --fail --retry-delay 1 --retry 30 --retry-connrefused http://localhost
+
+  run-dotrun:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dotrun
+        run: |
+          sudo snap install dotrun
+          sudo chown root:root /
+
+      - name: Install dependencies
+        run: dotrun install
+
+      - name: Build assets
+        run: dotrun build
+
+      - name: Test site
+        run: dotrun & curl --head --fail --retry-delay 1 --retry 30 --retry-connrefused http://localhost:8010
```

This should be:

```suggestion
        run: |
          dotrun &
          curl --head --fail --retry-delay 1 --retry 30 --retry-connrefused http://localhost:8010
```

Although `&` sounds like it's a conditional-like expression here, it really isn't. `&` is simply a special character to run a command in the background. What we're saying is "`dotrun &`: run dotrun in the background", and then later "`curl ...`: Check the server exists".